templates/CMakeLists.txt.template

```diff
@@ -196,18 +196,24 @@
         message(WARNING "gRPC_PROTOBUF_PROVIDER is \"module\" but PROTOBUF_ROOT_DIR is wrong")
     endif()
   elseif("<%text>${gRPC_PROTOBUF_PROVIDER}</%text>" STREQUAL "package")
-    find_package(protobuf CONFIG)
-    if(protobuf_FOUND)
+    find_package(Protobuf)
+    if(Protobuf_FOUND)
```

Comment:
Ok, right, I'm changing to caps, it seems that those variables are also exported in such form